docs/layer_configuration.md

```diff
@@ -0,0 +1,342 @@
+<!--
+Copyright 2023 The Khronos Group Inc.
+Copyright 2023 Valve Corporation
+Copyright 2023 LunarG, Inc.
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+[![Khronos Vulkan][1]][2]
+
+[1]: https://vulkan.lunarg.com/img/Vulkan_100px_Dec16.png "https://www.khronos.org/vulkan/"
+[2]: https://www.khronos.org/vulkan/
+
+# Layers Configuration
+
+[![Creative Commons][3]][4]
+
+[3]: https://i.creativecommons.org/l/by-nd/4.0/88x31.png "Creative Commons License"
+[4]: https://creativecommons.org/licenses/by-nd/4.0/
+
+Vulkan supports intercepting or hooking API entry points via a layer framework. A layer can intercept all or any subset of Vulkan API entry points. Multiple layers can be chained together to cascade their functionality in the appearance of a single, larger layer.
+
+Vulkan layers give Vulkan application developers the ability to add additional functionality to applications without modifying the application itself, e.g., validating API usages, dumping API entry points or generating screenshots of specified frames.
```

Comment:
Please (don't) laugh at my english remarks. I use grammarly. The suggestions are not necessarily better (until noted otherwise), use them only if you like them.

`add additional` maybe a bit of repetition. Alternative: _Vulkan layers allow application developers to add functionality to Vulkan applications without modifying the application itself_.

In the original sentence there could be a comma before "or".

Reply:
updated